cmd/backup/main.go

```diff
@@ -65,4 +82,79 @@ func main() {
 	s.must(s.withLabeledCommands(lifecyclePhaseProcess, s.encryptArchive)())
 	s.must(s.withLabeledCommands(lifecyclePhaseCopy, s.copyArchive)())
 	s.must(s.withLabeledCommands(lifecyclePhasePrune, s.pruneBackups)())
+
+	return nil
+}
+
+func main() {
+	serve := flag.Bool("foreground", false, "run the backup in the foreground")
+	envFolder := flag.String("env-folder", "/etc/dockervolumebackup/conf.d", "location of environment files")
+	flag.Parse()
+
+	logger := slog.New(slog.NewTextHandler(os.Stdout, nil))
+
+	if *serve {
+		logger.Info("running in the foreground instead of cron")
+
+		cr := cron.New()
+
+		addJob := func(c *Config) {
+			logger.Info("added cron job", "schedule", c.BackupCronExpression)
+			_, err := cr.AddFunc(c.BackupCronExpression, func() {
+				err := runBackup(c)
+				if err != nil {
+					logger.Error("unexpected error during backup", "error", err)
+				}
+			})
+			if err != nil {
+				logger.Error("failed to create cron job", "schedule", c.BackupCronExpression)
+			}
+		}
+
+		cs, err := loadEnvFiles(*envFolder)
+		if err != nil {
```

Comment:
This will need to be more specific. There are two types of errors `loadEnvFiles` can return:
- the directory simply does not exist (which is expected)
- something failed (e.g. permissions error or malformed content) when trying to read the files, which should make the script stop

The most straight forward thing to do would be to stat the directory in `loadEnvFiles` and return [`os.ErrNotExist`](https://pkg.go.dev/os#pkg-variables) in case it does not exist. `func main` can then check which error is returned and proceed accordingly.